app/views/pages/dashboard.html.erb

```diff
@@ -1,51 +1,185 @@
+        </li>
+      </ul>
+      <div class="tab-content" id="myTabContent">
+        <div class="tab-pane fade show active" id="favorite-developers" role="tabpanel" aria-labelledby="favorite-developers-list">
+          <h2>You don't have any developers saved to favorites</h2>
+          <%= link_to "View all developers", search_path %>
+        </div>
+        <div class="tab-pane fade" id="posted-jobs" role="tabpanel" aria-labelledby="posted-jobs-tab">
+          <ul>
+            <% @jobs.each do |job| %>
+              <div class="developer-card">
+                <% if job.company.photo.attached? %>
+                  <%= cl_image_tag application.developer.photo.key, height: 90, width: 90,  id: "dashboard-card-avatar" %>
+                <% else %>
+                  <%= image_tag "user.jpg", id: "dashboard-card-avatar" %>
+                <% end %>
+                <div class="developer-card-info">
+                  <div class="developer-card-left">
+                    <h2><%= job.job_title %></h2>
+                    <h3>Status: <%= job.active ? "Active" : "Closed" %></h3>
+                    <p><%= job.job_description.truncate(150) %></p>
+                  </div>
+                  <div class="developer-card-right">
+                    <h3>
+                      <% if (Date.today - job.created_at.to_date).to_i == 0 %>
+                        Posted today
+                      <% else %>
+                        Posted <%= (Date.today - job.created_at.to_date).to_i %> days ago
+                      <% end %>
+                    </h3>
+                    <div class="skill-tags">
+                      <!-- Skill tags code goes here -->
+                    </div>
+                  </div>
+                </div>
+              </div>
+            <% end %>
+          </ul>
+        </div>
+        <div class="tab-pane fade" id="received-applications" role="tabpanel" aria-labelledby="received-applications-tab">
+          <% @jobs.each do |job| %>
+            <% job.applications.each do |application| %>
+              <% if application.status == 'pending' %>
+                <%= link_to application_path(application) do %>
+                  <div class="developer-card">
+                    <% if application.developer.photo.attached? %>
+                      <%= cl_image_tag application.developer.photo.key, height: 90, width: 90,  id: "dashboard-card-avatar" %>
+                    <% else %>
+                      <%= image_tag "user.jpg", id: "dashboard-card-avatar" %>
+                    <% end %>
+                    <div class="developer-card-info">
+                      <div class="developer-card-left">
+                        <h2><%= application.developer.first_name.capitalize %> <%= application.developer.last_name.capitalize %></h2>
+                        <h3><%= application.developer.slogan.present? ? developer.slogan : "No title provided" %></h3>
```

Comment:
It crashed here. 

`undefined local variable or method `developer' for #<#<Class:0x00007f7bc9cd0748>:0x00007f7bc520e250>
Did you mean?  @developers`